iup/__init__.py

```diff
@@ -1010,21 +970,19 @@ def predict(
 
 
 #### prediction output ####
-class QuantileForecast(ValidatedUptake):
+class QuantileForecast(Data):
     """
     Class for forecast with quantiles.
     Save for future.
     """
 
-    def __init__(self, *args, **kwargs):
```

Comment:
Because `QuantileForecast` is a subclass of `Data`, it inherits any method in `Data`, including `__init__()`, that isn't overwritten

See https://github.com/CDCgov/cfa-immunization-uptake-projection/pull/65/commits/443f26fc326908e6342aa0a98f3ab75146f6f6c9 for a test that shows this works

(but also, I hadn't written a test, so I still had the validation as a list of tuples, rather than a dictionary. Good catch!)